lib/ProductOpener/DataQuality.pm

```diff
@@ -70,6 +69,7 @@ The type of product is specified through Config.pm
 
 package ProductOpener::DataQuality;
 
+use ProductOpener::PerlStandards;
 use utf8;
```

Comment:
We can remove:

use utf8;
use Modern::Perl '2017';

As their effect is included in ProductOpener::PerlStandards;